docs/extensions/services/working-with-users.md

```diff
@@ -0,0 +1,104 @@
+---
+description: Learn about the required services when working with users and granting them access control.
+contributors: Esther Agbaje
+---
+
+# Working with Users
+
+Directus provides a comprehensive set of services that facilitate seamless collaboration with users while enforcing
+access control to specific roles. They include:
```

Comment:
This reads very marketing to me. I would make it more straightforward. Something like: 

> There are various services related to users: